src/layout/ThemesSection/ThemesSection.svelte

```diff
@@ -16,38 +12,25 @@
 	let noInitialDelay = false;
 	let anchor: HTMLDivElement;
 
-	const themeColors = [
-		"var(--background-tertiary);",
-		"#414958",
-		"#6441a4",
-		"#feb400",
-		"#073642",
-		"#88c0d0"
+	const themeColors: Tag[] = [
+		{ name: "Dark Grey • Light Blue", color: "var(--fds-solid-background-tertiary)" },
+		{ name: "Blue Grey • Neon Green", color: "hsl(219, 15%, 30%)" },
+		{ name: "Purple", color: "hsl(261, 43%, 45%)" },
+		{ name: "Yellow • White", color: "hsl(43, 100%, 50%)" },
+		{ name: "Cool White • Dark Teal", color: "hsl(192, 81%, 14%)" },
+		{ name: "Sky Blue", color: "hsl(193, 43%, 67%)" }
```

Comment:
Was there a reason to choose an the bullet UTF character instead of a simply hyphen? Personally, I would have expected `&` or `-` more in that case.

Reply:
I find a bullet looks better than a dash when used as a separator. Totally personal preference though.

Reply:
The people have chosen the bullet for this, so be it.

Reply:
Well, bullets are better than ampersands, so that settles it.